models/engine/db_storage.py

```diff
@@ -40,6 +40,27 @@ def __init__(self):
         if HBNB_ENV == "test":
             Base.metadata.drop_all(self.__engine)
 
+    def get(self, cls, id):
+        """
+        Retrieve one object
+        @cls: class name
+        @id: string representing the object ID
+        Return: Object based on the class name and its ID, or None if not found
+        """
+        obj = self.__session.query(cls).get(id)
+        if obj is None:
+            return None
+        return obj
+
+    def count(self, cls=None):
+        """
+        Count the number of objects in storage:
+        @cls: class name
+        Return:
+        """
+        objs = self.all(cls)
```

Comment:
Incorporate this into the return statement for a super concise method

Reply:
Thank you !